services/app-web/src/pages/StateSubmission/StateSubmissionForm.tsx

```diff
@@ -199,6 +201,15 @@ export const StateSubmissionForm = (): React.ReactElement => {
 
             setFormDataFromLatestRevision(planFormData)
 
+            //set previous submitted files
+            const documentList = makeDocumentList(submissionAndRevisions)
+            if (documentList instanceof Error) {
+                //Maybe a different error message here.
+                setFormDataError('MALFORMATTED_DATA')
```

Comment:
Error state looks fine to me for now, I don't anticipate it will stay in this component since we are refactoring how this works.

FYI @rswerve this will be another thing to handle in our api wrapper that will isolate the data transformation.